### ISSUE TITLE
any way to run the new Docker Desktop onboarding tutorial post-install?

### ISSUE THREAD

**Issue by carehart** [docker/docker.github.io#10823]
File: [docker-for-windows/install.md](https://docs.docker.com/docker-for-windows/install/)
[This page in the Docker for Windows install docs](https://docs.docker.com/docker-for-windows/install/) mentions a newly available (2.3.0.2) "quick start tutorial" which will run after initial install, but it makes no mention of how the tutorial could be run otherwise/post-install, such as if one skips it (or especially for those who have long had DD installed but simply upgrade to the version adding the tutorial).

I have looked in the DD menu, and even the new dashboard (added in 2.2.0.3), but I don't find any way to re-launch the tutorial. 

So I am asking first if it is possible to do it, then besides sharing it here, I hope someone would add it to the docs here (and sure, I could)..

BTW, the tutorial is more than just a another how-to, but (according to [this docker.com blog post on it](https://www.docker.com/blog/a-new-way-to-get-started-with-docker/)) shows an interactive terminal allowing entry of commands. It would be helpful (even for others, with DD already installed) to be able to at least experience it, to share the news with others.

And FWIW, the post also makes no mention of how to launch it post-install, and it would be nice to see this clarified there also (but it also offers no feedback mechanism for me to ask this there).

**Comment by StefanScherer** [comment]
Thank you @carehart for the feedback. Good to hear that you like the tutorial. 
Indeed there is a way to run the tutorial again. Open the DD menu and click on `Learn` and this will start the tutorial again. 
May I ask you what you have expected to find it in the DD menu more easily?

**Comment by carehart** [comment]
Thanks. So first, I meant to try that (and should have, before writing) but at first I presumed it would go to some other form of "learning". Perhaps if it said "quick start tutorial", that would have tied better to what the docs and blog post say, as well as to be more specific (because it's not really a link for "learning", beyond this one tutorial, live and interactive though it is.)

I have just [posted a PR](https://github.com/docker/docker.github.io/pull/10830) to add mention of this to the docs. As for that blog post, again I find no ready way to get a similar proposal to the author. 

All that said, on the tutorial itself, I was surprised it ran a 21-step build. Also, it didn't end proposing that folks launch what they'd built. Then when I did launch that, it referred to "the command you just ran" which did not match what had been run per that tutorial. 

Where would be the best place to bring up feedback like that? Simply here as a new issue, or any place more particular? Thanks.
